slike/fenko.tikz

```diff
@@ -3,30 +3,30 @@
 \tikzstyle{dec}=[draw,rectangle,minimum size=0.6cm]
 \tikzstyle{ran}=[draw,circle,minimum size=0.6cm]
 \tikzstyle{pick}=[double, very thick]
-\node[dec] (A) at (-8,-2) [label=left:$7.65$ G] {};
-\node[ran] (B) at (-4, 2) [label=135:$7.65$ G] {};
-\node[ran] (C) at (-4,-6) [label=225:$5.68$ G] {};
+\node[dec] (A) at (-8,-2) [label=left:$5.84$ G] {};
+\node[ran] (B) at (-4, 2) [label=left:$5.84$ G] {};
+\node[ran] (C) at (-4,-6) [label=left:$5.68$ G] {};
 \node      (D) at (-4,-2) {$5$ G};
-\node[dec] (E) at ( 0, 4) [label=225:$9$ G] {};
-\node[dec] (F) at ( 0, 0) [label=225:$3$ G] {};
-\node[dec] (G) at ( 0,-4) [label=225:$7$ G] {};
-\node[dec] (H) at ( 0, -8) [label=225:$3$ G] {};
-\node[ran] (I) at ( 4, 5) [label=225:$14.33$ G] {};
+\node[dec] (E) at ( 0, 4) [label=left:$9$ G, kupi $4$ kg] {};
+\node[dec] (F) at ( 0, 0) [label=left:$3$ G, kupi $1$ kg] {};
+\node[dec] (G) at ( 0,-4) [label=left:$7$ G, kupi $3$ kg] {};
+\node[dec] (H) at ( 0, -8) [label=left:$3$ G, kupi $2$ kg] {};
```

Comment:
Kolikor vidim, je problem tukaj - Ti*k*Z razume vejico kot ločilo polj, zato pride do napake (če bi jo hotel imeti kot oznako vozlišča, bi moral vse skupaj oviti v zavite oklepaje). Sicer pa to ni mesto za pisanje odločitev - v oznakah vozlišč naj bodo pričakovane vrednosti v teh vozliščih (ker gre za odločitvena vozlišča, bo torej oznaka enaka kot pri izbrani veji).

Mislim tudi, da bo vendarle izgledalo lepše, če so te oznake nad ali pod vozlišči (enako tudi pri slučajnih dogodkih za drugo noč).